docs/en/index.md

```diff
@@ -45,6 +45,12 @@ Example router might look like this
 $router[] = new Route('[<locale=cs cs|en>/]<presenter>/<action>', "Homepage:default");
 ```
 
+or with new version of Nette, you can get localization by TLD (top level domain) look like this
+
+```php
+$router[] = new Route('//%sld%.<locale=cs cs|sk>/<presenter>/<action>', "Homepage:default");
```

Comment:
I like the example, it would work nice for `sk`, `de` but definitely not for US_en or CS_cz.

If you figure out how to make it simple, I'd be happy to merge it.